crates/task-impls/src/quorum_vote.rs

```diff
@@ -143,53 +306,38 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES> + 'static> HandleDepOutput
         )
         .await;
 
-        if !self.quorum_membership.has_stake(&self.public_key) {
-            debug!(
-                "We were not chosen for quorum committee on {:?}",
+        let Some(vid_share) = vid_share else {
```

Comment:
Same deal here, how can we reach this code with vid_share being `None`

Reply:
We can't get here for this and below the idea is just that we needed some way to get the variables out of the big match statement one at a time. This is an admittedly less glamorous way to do it, but we should be good.

Reply:
I see, I think it's fine then.